portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function showInfo(number) {
+  var elementId = "info-" + number;
+  console.log(elementId);
+  var infoDiv = document.getElementById(elementId);
+  var buttonDiv = document.getElementById("div-" + number);
+
+  console.log("Button has been pressed. Display of element is " + infoDiv.style.display);
+  
+  if (infoDiv.style.display === "none") {
+    infoDiv.style.display = "block";
+    buttonDiv.style.display = "none";
+  }
 }
+
+function showGreeting() {
```

Comment:
is there a reason you didn't reuse `addRandomGreeting` here?

Reply:
It is a function from the tutorial I forgot to remove from my script. I've removed it now.